src/nacl/pwhash/__init__.py

```diff
@@ -63,7 +63,6 @@ def verify(password_hash, password):
     in the stored hash
     """
     _decoded_hash = password_hash.decode('ascii')
-    # the bytes.beginswith() method isn't supported by python 3.3
```

Comment:
I think you fixed the wrong thing :-) We can either leave the comment, or switch to stop decoding everything here.

Reply:
@alex is right. I think it would be better to leave the comment alone for now and add a subsequent commit for replacing decoded hash/decoded strprefix in the dispatcher with straight
```
if  password_hash.startswith(argon2id.STRPREFIX):
     . . .
```
etc.

Reply:
Oops, didn't mean to remove this 😄